packages/context/src/objects/objects.ts

```diff
@@ -43,6 +43,37 @@ export type ObjectInstance = {
     [key: string]: unknown;
 };
 
+export type FormioInputTypes = {
+    label: string;
+    type: string;
+    key: string;
+    initialValue?: any;
+    property?: Property;
+    input?: boolean;
+    placeholder?: string;
+    description?: string;
+    tooltip?: string;
+    prefix?: string;
+    suffix?: string;
+    showCharCount?: boolean;
+    readOnly?: boolean;
+    isMultiLineText?: boolean;
+    widget?: string;
```

Comment:
What are `widget`, `tableView`, `verticalLayout`, `data`, and `dataSrc` for?

Reply:
The `data` contains the values for the `select` options. The `dataSrc` is referencing where to get the values from i.e. `dataSrc: "values"`.
The `verticalLayout` prop will be used for section layout if we want a vertical layout or not. It's hard-coded. There's no way to set the `verticalLayout` and `tableView` in the builder and I believe this could be removed.

Reply:
I tried changing the `dataSrc` to some random string but the formio still getting the values from `data.values`. This can also be removed.